src/components/shared/Extensions/Accordion/Accordion.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { FC } from 'react';
-import AccordionItem from './partials/AccordionItem';
+import AccordionItem from './Partials/AccordionItem';
```

Comment:
`partials` should be lowercase because it's a directory (like `common`, `shared`, `hooks` etc), not component,
please, fix it in every your component